app/guide/interfaces.vmd

```diff
@@ -137,7 +137,7 @@ The current syntax is closer to the way Vale internally thinks about interfaces.
  * We could have override methods inside the `impl`.
 
 ////
-```
+```Vale
```

Comment:
Leaving this off was actually intentional, since Vale doesn't understand this simplified syntax =\

Reply:
ok, good reason